src/Impostor.Api/Innersloth/GameOptionsData.cs

```diff
@@ -1,59 +1,157 @@
 
+        /// <summary>
+        /// Gets or sets the maximum amount of emergency meetings each player can call during the game in seconds.
+        /// </summary>
         public int NumEmergencyMeetings { get; set; }
 
+        /// <summary>
+        /// Gets or sets the cooldown between each time any player can call an emergency meeting in seconds.
+        /// </summary>
         public int EmergencyCooldown { get; set; }
 
+        /// <summary>
+        /// Gets or sets the number of impostors for this lobby.
+        /// </summary>
         public int NumImpostors { get; set; }
 
+        /// <summary>
+        /// Gets or sets a value indicating whether ghosts (dead crew members) can do tasks.
+        /// </summary>
         public bool GhostsDoTasks { get; set; }
 
-        public int KillDistance { get; set; }
-
+        /// <summary>
+        /// Gets or sets the Kill as per values in <see cref="KillDistances"/>.
+        /// </summary>
+        /// <remarks>
+        /// Short = 0, Normal = 1, Long = 2.
+        /// </remarks>
+        public KillDistances KillDistance { get; set; }
+
+        /// <summary>
+        /// Gets or sets the time for discussion before voting time in seconds.
+        /// </summary>
         public int DiscussionTime { get; set; }
 
+        /// <summary>
+        /// Gets or sets the time for voting in seconds.
+        /// </summary>
         public int VotingTime { get; set; }
 
+        /// <summary>
+        /// Gets or sets a value indicating whether an ejected player is an impostor or not.
+        /// </summary>
         public bool ConfirmImpostor { get; set; }
 
+        /// <summary>
+        /// Gets or sets a value indicating whether players are able to see tasks being performed by other players.
+        /// </summary>
+        /// <remarks>
+        /// By being set to true, tasks such as Empty Garbage, Submit Scan, Clear asteroids, Prime shields execution will be visible to other players.
+        /// </remarks>
         public bool VisualTasks { get; set; }
 
+        /// <summary>
+        /// Gets or sets a value indicating whether the vote is anonymous.
+        /// </summary>
         public bool AnonymousVotes { get; set; }
 
+        /// <summary>
+        /// Gets or sets the task bar update mode as per values in <see cref="Innersloth.TaskBarUpdate"/>.
+        /// </summary>
         public TaskBarUpdate TaskBarUpdate { get; set; }
 
+        /// <summary>
+        /// Gets or sets a value indicating whether the GameOptions are the default ones.
+        /// </summary>
         public bool IsDefaults { get; set; }
 
+        /// <summary>
+        /// Deserialize a packet/message to a new GameOptionsData object.
+        /// </summary>
+        /// <param name="reader">Message reader object containing the raw message.</param>
+        /// <returns>GameOptionsData object.</returns>
+        public static GameOptionsData DeserializeCreate(IMessageReader reader)
```

Comment:
this method was moved up just because VS said to have static methods first in the class